src/mod/server.mod/servmsg.c

```diff
@@ -1412,6 +1431,19 @@ static int handle_sasl_timeout()
   return sasl_error("timeout");
 }
 
+/* Got AWAY message; only valid for IRCv3 away-notify capability */
+static int gotawayv3(char *from, char *msg)
+{
+  if (strlen(msg)) {
+    fixcolon(msg);
+    putlog(LOG_SERV, "*", "%s is now away: %s", from, msg);
```

Comment:
I believe this should be LOG_JOIN | LOG_MISC just like the "%s joined %s" message for joins